includes/misc/nginx/nginx.conf

```diff
@@ -30,7 +30,7 @@ http {
 	include /etc/nginx/mime.types;
 	default_type application/octet-stream;
 	access_log off;
-	error_log /dev/null;
+	error_log /dev/logs;
```

Comment:
Shouldn't logs got to `stdout`?

Reply:
actually earlier all the logs were discarded (piped to /dev/null), but I put them to the logs so that they can be viewed later on also